src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
 spring.application.name=BookStoreMate
+spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
+spring.datasource.username=root
+spring.datasource.password=1234
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.show-sql=true
```

Comment:
each file should end with exactly one empty line, otherwise this will be shown on GitHub
![image](https://github.com/Shr1mpa/BookStore/assets/49767049/59081bf2-4cda-41c3-9c12-ace8164cd582)